cumulus/constants.py

```diff
@@ -54,3 +60,20 @@ class JobQueueState:
     RUNNING = 'running'
     COMPLETE = 'complete'
     ERROR = 'error'
+
+
+class ClusterStatus(IntEnum):
+    error = -1
+    creating = 0
+    created = 10
+    launching = 20
+    launched = 30
+    provisioning = 40
+    provisioned = 50
+    terminating = 60
+    terminated = 70
+    stopped = 101
+    running = 102
+
```

Comment:
If we are going to move toward using enum for the states we should do the same for job etc. in the future.